.gitignore

```diff
@@ -1,3 +1,6 @@
 .env
 .idea/
 *.csv
+
```

Comment:
did you run go mod vendor? you shouldn't need to ever vendor in this repo

Reply:
i might've back when i was exploring go stuff